### PR TITLE
Initial connection

### PR DESCRIPTION
**Pull request by beneggett**

### CODE REVIEW

File: lib/orbital/gateway.rb [hunk 1]
```diff
@@ -1,4 +1,18 @@
+require "active_support/all"
+require "nokogiri"
+require "builder"
+require "httparty"
+require "ostruct"
+
```
Comment (beneggett) [inline lib/orbital/gateway.rb:6]: Are we still using all these dependencies?

File: lib/orbital/gateway/api.rb [hunk 2]
```diff
@@ -0,0 +1,141 @@
+module Orbital
+  module Gateway
+    class Api
+      URL_ENDPOINT = ENV.fetch("ORBITAL_GATEWAY_BASE_URL", "https://orbitalvar1.chasepaymentech.com/authorize")
```
Comment (beneggett) [inline lib/orbital/gateway/api.rb]: Technically chase has 2 end points, 1 live & a failover; we may try and build in some failover logic to try the other one  for production

File: lib/orbital/gateway/api.rb [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      API_VERSION = ENV.fetch("ORBITAL_GATEWAY_API_VERSION", "7.4")
+      ORBITAL_MERCHANT_ID = ENV.fetch("ORBITAL_MERCHANT_ID", "not-configured")
+      ORBITAL_CONNECTION_ID = ENV.fetch("ORBITAL_CONNECTION_ID", "not-configured")
+      ORBITAL_CONNECTION_PASSWORD = ENV.fetch("ORBITAL_CONNECTION_PASSWORD", "not-configured")
+      BIN = ENV.fetch("ORBITAL_BIN", "stratus")
+      include HTTParty
+
+      def initialize(options = {})
+        # @username = options[:username] || ENV["PPG_USERNAME"]
```
Comment (beneggett) [inline lib/orbital/gateway/api.rb]: we can remove a bunch of comments

File: 2 [hunk 4]
```diff
@@ -0,0 +1,289 @@
+require "test_helper"
```
Comment (phoffer) [inline 2]: did this file pop in by accident?

File: test/orbital/gateway/api/authorization_test.rb [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+class Orbital::Gateway::Api::AuthorizationTest < Minitest::Test
+  describe "Auth only tests" do
+
+    let(:gateway) { Orbital::Gateway::Api::Authorization.new }
+
+    it 'tests created profile amex 5.1' do
+      # skip "TODO Failing.  Need to look into why"
+      body = gateway.auth_only(amex_params.merge({amount: '1'}))
+      result = gateway.post(body)
+      binding.pry
```
Comment (phoffer) [inline test/orbital/gateway/api/authorization_test.rb]: can these all be removed?

**Pull request merged**